astropy/io/ascii/core.py

```diff
@@ -671,7 +671,7 @@ class BaseData(object):
     fill_include_names = None
     fill_exclude_names = None
     # Currently, the default matches the numpy default for masked values.
```

Comment:
I'm not opposed to this change, I just wanted to make sure it's a conscious decision to move away from the numpy format since that was the reason for having -- in the first place.